ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/common/util/ForkChoiceUtilTest.java

```diff
@@ -182,22 +183,32 @@ public void getSlotStartTime_shouldGetCorrectTimePastGenesis() {
 
   @Test
   void onTick_shouldExitImmediatelyWhenCurrentTimeIsBeforeGenesisTime() {
-    final MutableStore store = mock(MutableStore.class);
+    final MutableStore store = spy(MutableStore.class);
     when(store.getGenesisTime()).thenReturn(UInt64.valueOf(3000));
-    when(store.getTimeSeconds()).thenReturn(UInt64.ZERO);
-    forkChoiceUtil.onTick(store, UInt64.valueOf(2000));
+    when(store.getTimeMillis()).thenReturn(UInt64.ZERO);
+    forkChoiceUtil.onTick(store, UInt64.valueOf(2000000));
 
-    verify(store, never()).setTimeSeconds(any());
+    verify(store, never()).setTimeMillis(any());
   }
 
   @Test
   void onTick_shouldExitImmediatelyWhenCurrentTimeIsBeforeStoreTime() {
-    final MutableStore store = mock(MutableStore.class);
+    final MutableStore store = spy(MutableStore.class);
     when(store.getGenesisTime()).thenReturn(UInt64.valueOf(3000));
-    when(store.getTimeSeconds()).thenReturn(UInt64.valueOf(5000));
-    forkChoiceUtil.onTick(store, UInt64.valueOf(4000));
+    when(store.getTimeMillis()).thenReturn(UInt64.valueOf(5000000));
+    forkChoiceUtil.onTick(store, UInt64.valueOf(4000000));
 
-    verify(store, never()).setTimeSeconds(any());
+    verify(store, never()).setTimeMillis(any());
+  }
+
+  @Test
+  void onTick_setsStoreTimeAndStoreTimeMillis() {
+    final MutableStore store = spy(MutableStore.class);
+    when(store.getGenesisTime()).thenReturn(UInt64.valueOf(3000));
+    when(store.getTimeMillis()).thenReturn(UInt64.valueOf(4000000));
+    forkChoiceUtil.onTick(store, UInt64.valueOf(5000678));
+
+    verify(store).setTimeMillis(UInt64.valueOf(5000678));
```

Comment:
I think I'd just move this one to using a stub:
```suggestion
    final TestStoreImpl store = new TestStoreFactory(spec).createGenesisStore();
    final UInt64 expectedMillis = store.getTimeMillis().plus(123456);
    forkChoiceUtil.onTick(store, expectedMillis);

    assertThat(store.getTimeMillis()).isEqualTo(expectedMillis);
```

And maybe the test name is just `onTick_setsStoreTimeMillis`?